scene/debugger/scene_debugger.h

```diff
@@ -31,17 +31,13 @@
 #ifndef SCENE_DEBUGGER_H
 #define SCENE_DEBUGGER_H
 
-#include "core/input/shortcut.h"
-#include "core/object/ref_counted.h"
-#include "core/string/ustring.h"
-#include "core/templates/pair.h"
-#include "core/variant/array.h"
 #include "scene/gui/view_panner.h"
```

Comment:
This should be working with RefCounted, but for whatever reason I can't forward-declare ViewPanner here.